src/results.cpp

```diff
@@ -230,11 +242,12 @@ void Results::evaluate_query_if_needed(bool wants_notifications)
         return;
     }
 
-    if (m_notifier)
+    // if (m_notifier) FIXME: Why was this condition added?
```

Comment:
This looks like it was just a stray paste and should be deleted.

Reply:
Done